libs/ledgerjs/packages/cryptoassets/README.md

```diff
@@ -130,14 +126,6 @@ Returns **(CryptoCurrency | null | [undefined](https://developer.mozilla.org/doc
 
 Returns **(CryptoCurrency | null | [undefined](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/undefined))** 
 
-### findCryptoCurrencyById
-
-#### Parameters
-
-*   `id` **any** 
-
-Returns **(CryptoCurrency | null | [undefined](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/undefined))** 
-
```

Comment:
Doesn't seem normal that those functions are being removed from the doc, especially in this PR.

Reply:
I ran `pnpm doc:ljs` it's not a manual update

Reply:
Yes because you removed that function, and you shouldn't.

Reply:
I know but it still shouldn't be happening so there is an issue somewhere.